test/framework/module_generator.py

```diff
@@ -115,8 +115,8 @@ def test_descr(self):
                 "    }",
                 "}",
                 '',
-                "module-whatis {foo}",
-                "module-whatis {bar}",
+                "module-whatis {Description: foo}",
+                "module-whatis {Description: bar}",
```

Comment:
the `Description: ` part is not there is `whatis` is specified, so change this back to what it was